resources/Microsoft.Windows.Setting.Time/Microsoft.Windows.Setting.Time.psm1

```diff
@@ -0,0 +1,235 @@
+$global:tzAutoUpdatePath = "HKLM:\SYSTEM\CurrentControlSet\Services\W32Time\Parameters"
```

Comment:
Is there a reason for not initializing these like the registry paths are initialized in `Microsoft.Windows.Accessibility` so that tests don’t use the real registry?

Reply:
Please see the comment on #93. Pester docs: https://pester.dev/docs/usage/testregistry

Reply:
Interesting. . .